.github/workflows/rasa-tests.yml

```diff
@@ -0,0 +1,103 @@
+name: Rasa CI
+
+on:
+  # Trigger analysis when pushing in master or pull requests, and when creating
+  # a pull request.
+  push:
+    branches:
+      - rasa_testing
+      - main
+  pull_request:
+      types: [opened, synchronize, reopened]
+
+jobs:
+  type-testing:
+    name: Test Types
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r Rasa_Bot/requirements_dev.txt
+    - name: Type Checking
+      working-directory: ${{ github.workspace }}/Rasa_Bot
+      run: |
+        pip list
+        make types
+  data-validation:
+    name: Validate Data
+    runs-on: ubuntu-latest
+    env:
+      working-directory: ./Rasa_Bot
+    steps:
+    - uses: actions/checkout@v1
+    - name: Set up Python 3.7
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.7
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip
+        pip install -r Rasa_Bot/requirements.txt
+    - name: Rasa Data Validation
+      run: | 
+        cd Rasa_Bot
+        rasa telemetry disable
+        rasa data validate --max-history 10
```

Comment:
Ideally we should get the 10 here and in config.yml from somewhere. (nice to have) -> create an issue. Also a problem with the rasa version -> single version file?